apis/workloads/v1alpha1/lifecycle_recorder.go

```diff
@@ -208,3 +221,38 @@ func (l *lifecycleEventRecorderImpl) ConsoleTerminate(ctx context.Context, csl *
 	l.logger.Info("event recorded", "id", id, "event", events.EventTerminated)
 	return nil
 }
+
+func appendStatusMessages(result map[string]string, containers []corev1.ContainerStatus) {
```

Comment:
```suggestion
func appendStatusMessages(result map[string]string, containerStatuses []corev1.ContainerStatus) {
```

Nit: Maybe this fits more naturally?

Reply:
Yes totally! I will change it.